src/app/content/components/popUp/FiltersList.tsx

```diff
@@ -88,9 +89,9 @@ interface FiltersListChapterProps {
 export const FiltersListChapter = (props: FiltersListChapterProps) => (
   <FilterListItem>
     <StyledPlainButton
-      aria-label={useIntl().formatMessage({
-        id: 'i18n:highlighting:filters:remove:chapter'},
-        {filterValue: splitTitleParts(props.title).join(' ')}
+      aria-label={useIntl().formatMessage(
+        { id: props.ariaLabelKey },
+        { filterValue: splitTitleParts(props.title).join(' ') }
       )}
       data-analytics-label={`Remove breadcrumb for chapter ${splitTitleParts(props.title).join(' ')}`}
```

Comment:
I think this analytics label and the second one for `FiltersListColor` should be generated in 
`src/app/content/highlights/components/SummaryPopup/Filters.tsx` and
`src/app/content/studyGuides/components/Filters.tsx `